src/inverseRobot.js

```diff
@@ -8,6 +8,23 @@
 
 function inverseRobot(robot) {
   // write code here
+
+  const reversed = {};
+  const check = {};
```

Comment:
we don't really need `check` object - after each loop iteration values will be updated and then we can check if property already exists